digitalpy/core/telemetry/meter.py

```diff
@@ -11,7 +11,13 @@
 
 class Meter(ABC):
         
-    def __init__(self, provider: MetricsProvider, meter_name=None):
+    def __init__(self, provider: MetricsProvider, meter_name: str=None):
+        """the constructor for the abstract meter class
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D401 First line should be in imperative mood; try rephrasing